app/assets/javascripts/likes.js

```diff
@@ -0,0 +1,12 @@
+$(document).ready(function() {
+
+  $('.likes-link').on('click', function(event){
+      event.preventDefault();
+
+      var likeCount = $(this).siblings('.likes_count');
+
+      $.post(this.href, function(response){
+        likeCount.text(response.new_like_count);
+    })
+  })
+})
```

Comment:
You could handle this in a Rails-y way. You can use a JavaScript view that takes JSON, and executes JS in the response. Check [here](http://guides.rubyonrails.org/working_with_javascript_in_rails.html) for more.